### DISCUSSION TITLE
Reset zoom center

### DISCUSSION THREAD

**Discussion by Ofboir** [osmandapp/OsmAnd#21466] [Q&A] [answered]
Hi

The center for zoom recently changed in my Osmand : it used to be on the center of the screen, but it is now towards the bottom (you can see it on the picture).
![osmand](https://github.com/user-attachments/assets/b8dafa1d-9853-42ae-9a0a-9b3dcd21139b)

I probably did something wrong, but I don't know how to reset it. How could I do ?
Thank you

**Comment by vshcherb** [comment]
It's a setting to change center of the screen Android settings

**Reply by Ofboir** [reply]
Hi, thank you for your answer !
What do you mean ? Is it in Android settings and not in Osmand settings ? I didn't find it.

**Reply by vshcherb** [reply, marked as answer]
Configure Screen -> Location position - [Bottom, Center]

**Reply by Ofboir** [reply]
Ok, that's it !
I had tried it before, but what I didn't know is that **you need to restart the application to see the change** !
Thank you vshcherb !
